stylesheets/_font_stack.scss

```diff
@@ -35,10 +27,7 @@ $NTA-Light-Tabular:
        local("Helvetica");
 }
 
-$Helvetica-Regular:
-  "GDS-Logo",
-  sans-serif;
+$Helvetica-Regular: "GDS-Logo", sans-serif;
```

Comment:
@gemmaleigh I think the fonts aren't in here deliberately, to avoid licensing issues. I've just had a discussion about this on slack: https://ukgovernmentdigital.slack.com/archives/frontend/p1448023249000062